### PR TITLE
Format and tidy up user guide

### PR DESCRIPTION
**Pull request by nananakx-x**
-Made names case-sensitive when detecting the adding of duplicate fosterers and also let it ignore the presence of multiple spaces between words

### CODE REVIEW

File: src/main/java/seedu/address/model/person/Name.java [hunk 1]
```diff
@@ -56,7 +56,7 @@ public boolean equals(Object other) {
         }
 
         Name otherName = (Name) other;
-        return fullName.equals(otherName.fullName);
+        return fullName.toUpperCase().equals(otherName.fullName.toUpperCase());
```
Comment (butteredyakiimo) [inline src/main/java/seedu/address/model/person/Name.java]: I think we haven't covered the case where there is an extra space between the names :O eg: john doe and john      doe will still be detected as the same person.
Screenshot 2023-11-01 at 8 55 59 AM
we can either add this part under planned enhancements to warn them about the space, or check using hashcode :P

File: docs/UserGuide.md [hunk 2]
```diff
@@ -127,195 +125,256 @@ Valid cases:
 | 4   | Not fostering (e.g. overseas, currently not able to foster) | `NotAvailable`            | `nil`             | `nil`         |
 | 5   | Fostering: ALL information must be present                  | `NotAvailable`            | `current.Dog/Cat` | NOT `nil`          |
 
+* For invalid cases, error messages will be shown when the invalid command is entered. For example:
+  ![Add](images/screenshots/AddErrorMessage.png)
+
+<div markdown="span" class="alert alert-primary">
+  :exclamation: <b>Important:</b> <br/>
+Duplicate fosterers cannot be added; this is detected by the fosterer's name.</br>
+e.g. <code>"Anne Tay"</code> is the same person as <code>"anne tay"</code>.
```
Comment (butteredyakiimo) [inline docs/UserGuide.md]: If you're not planning to resolve the blank space issue, can maybe put a warning here too! + in the planned enhancements

File: docs/UserGuide.md [hunk 3]
```diff
@@ -127,195 +125,256 @@ Valid cases:
...
+
+In the case where duplicates field descriptions are given, the last one will be chosen:
+* `add n/Jerry Tan p/98765412 e/jerry123@example.com a/Baker street, block 5, #27-01 housing/HDB housing/Condo availability/Available availability/NotAvailable animal/Dexter animal/Happy animalType/able.Dog animalType/current.Cat t/Urgent`
+  * adds a fosterer named Jerry Tan with the phone number 98765412 and email address jerry123@example.com; his address is Baker street, block 5, #27-01, housing type is Condo and he is fostering a cat named Dexter. An urgent visit is required.
+
+![Add](images/screenshots/AddSuccess.png)
+
+### Adding a fosterer through the profile page: `add`
+
+Adds a fosterer to the address book with information provided through the profile page.
```
Comment (butteredyakiimo) [inline docs/UserGuide.md]: maybe can change to: Opens up an empty profile page, where details of the fosterer can be filled in. Adds the fosterer to the address book upon saving the information.

Comment (butteredyakiimo) [reply docs/UserGuide.md]: I think what we can do to reduce the duplication of the information in the add and edit using the profile screen will be to link a separate section called `Navigating the Profile Screen`, and then including stuff like field jumping, save and exit. @brandon-nam maybe you can take over that? In this section, can include both the links back to the add and edit command for convenience.
I think this section can be put at the front before the commands, but whichever is more intuitive :P

Then under this add section, can just include a screenshot of the empty profile screen, and then just say refer to that section + put the link there + mention how its with the same validity checks as the normal add.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -127,195 +125,256 @@ Valid cases:
...
+
+### Listing fosterers: `list` (Alias: `find`)
 
 Lists fosterers that match a particular description or search, or all fosterers if the search is blank.
 
 Format: `list *KEYWORDS`
 
 Alias: `find`
 
+<div markdown="block" class="alert alert-info">
```
Comment (butteredyakiimo) [inline docs/UserGuide.md:171]: actually, why is the whole thing wrapped in the information source box

File: docs/UserGuide.md [hunk 5]
```diff
@@ -127,195 +125,256 @@ Valid cases:
...
+</div>
+
+Examples:
+* `list` 
+  * lists all fosterers in the address book.
+* `list john doe`
+  * lists entries which match "John Doe", "Doe John", "Johnny Doe", and "Mary" who lives on "John Doe Street".
+* `find john john doe` 
+  * is redundant and gives the same result as `find john doe`.
+* `list "John" | zam & doe`
```
Comment (butteredyakiimo) [inline docs/UserGuide.md:211]: Leaving this to ZH to update the need to add the " "

File: docs/UserGuide.md [hunk 6]
```diff
@@ -127,195 +125,256 @@ Valid cases:
...
+Only specific commands are available in the profile page. 
+The list of available commands are <code>help</code>, <code>save</code>, and <code>exit</code>. 
 </div> 
 
 Examples:
-* `list` followed by `view 2` to view the profile of the 2nd fosterer in the address book.
+* `list` followed by `view 2`
+  * views the profile of the 2nd fosterer in the address book.
 
 =======
```
Comment (butteredyakiimo) [inline docs/UserGuide.md:236]: need to delete this HAHA

File: docs/UserGuide.md [hunk 7]
```diff
@@ -127,195 +125,256 @@ Valid cases:
...
+**:information_source: Notes about the command format:**<br>
+
+*  The UI generates a profile page where details of the fosterer are shown.
+*  Aside from the details added by the add command, here is the list of fields the profile page shows:
+* `NOTE` : Notes for the user, examples include but are not limited to (optional and non-exhaustive):
+  * Health condition of the animal
+  * Duration / end date of foster, whichever the foster manager prefers
+  * Identifiable physical traits of the animal
+*  The respective updated field descriptions can be keyed in and saved by the user through the `save` command.
+  * Please refer to the section 'Viewing a fosterer's detail: Editing a fosterer’s details in profile page' for the specific steps to do so.
```
Comment (butteredyakiimo) [inline docs/UserGuide.md:337]: You mean edit using profile view here right! cant find the section 'Viewing a fosterer's detail: Editing a fosterer’s details in profile page'

**Review by butteredyakiimo — COMMENTED** [review]
I only read through briefly, as the specifics in the profile view has yet to be updated. will read through again once everyone has updated their own parts + included a suggestion on how to deal with repeating info for the edit and add command!

File: src/main/java/seedu/address/model/person/Name.java (hunk 1 above)
Comment (nananakx-x) [reply src/main/java/seedu/address/model/person/Name.java]: for now, I just use the same warning for duplicate person!

File: docs/UserGuide.md (hunk 6 above)
Comment (nananakx-x) [reply docs/UserGuide.md:236]: Wait why?

File: docs/UserGuide.md (hunk 7 above)
Comment (nananakx-x) [reply docs/UserGuide.md:337]: Its correct...? so confused HAHAHA its the subsection under view feature which brandon created

File: docs/UserGuide.md (hunk 2 above)
Comment (nananakx-x) [reply docs/UserGuide.md]: included!

**Review by butteredyakiimo — APPROVED** [review]

**Pull request merged**